package.json

```diff
@@ -24,16 +24,17 @@
 		"last 2 Opera versions"
 	],
 	"devDependencies": {
+		"@lodder/grunt-postcss": "^3.1.1",
```

Comment:
Does this negate the need for `grunt-postcss` or is that different?

Reply:
Ah! Yes, good find. `grunt-postcss` is [archived and considered abandoned](https://github.com/nDmitry/grunt-postcss). The [PostCSS release notes includes this as the suggested replacement](https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users#grunt).